src/components/study-container.js

```diff
@@ -248,11 +252,8 @@ export function StudyContainer({ view, onChangeTab }) {
     // using a ref because this is not used for rendering, it is used in the websocket onMessage()
     const studyParentDirectoriesUuidsRef = useRef([]);
 
-    const network = useSelector((state) => state.network);
     const userName = useSelector((state) => state.user.profile.sub);
     const paramsLoaded = useSelector((state) => state[PARAMS_LOADED]);
-    const [networkLoadingFailMessage, setNetworkLoadingFailMessage] =
```

Comment:
You don't manage the loading fail of equipments ?
Now the app is stuck on "loading" if I shut down a service or if we receive an error.

Reply:
fixed